odk1-src/form-styling.rst

```diff
@@ -0,0 +1,307 @@
+.. spelling::
+
+  CSS
+  bolding
+  monospace
+  tt
+  yn
+	
+Form Styling
+==============
+
+Labels, hints, and choices in an :doc:`xlsform`
+can all be styled using 
+:ref:`markdown-in-forms`, :ref:`html-in-forms`, and :ref:`emoji`.
+
+
+.. _markdown-in-forms:
+
+Markdown
+---------
+
+:doc:`xlsform` supports limited used of `Markdown`_.
+
+.. _Markdown: https://en.wikipedia.org/wiki/Markdown
+
+.. _markdown-headers:
+
+Headers
+~~~~~~~~
+
+Labels and hints can be styled with one of six header levels.
+
+.. code-block:: none
+
+  # Header H1
+  
+  ## Header H2
+  
+  ### Header H3
+  
+  #### Header H4
+  
+  ##### Header H5
+  
+  ###### Header H6
+
+    
+.. image:: /img/form-styling/h1-label.* 
```

Comment:
It's fine to have the various headers, but the multiple screenshots don't add much. It's a lot of scrolling! Maybe we just have H1 and H6 and people can imagine what H2/3/4/5 look like?

Reply:
Alternatively, H2-H5 could be hidden under show detail?

Reply:
Yeah. Something in an earlier comment from you (can't find now) made me think you were suggesting this. I originally wanted to put all six on one screen, but that doesn't actually work.
So --- if you think we don't really need to see all six sizes rendered, I'll pull them out.

Reply:
One option is to put them all in a note, which gives you a sense of the size.